mods/taskbar-clock-customization.wh.cpp

```diff
@@ -876,18 +881,6 @@ std::wstring EscapeUrlComponent(PCWSTR input,
     return out;
 }
 
-std::wstring GetWeatherCacheKey() {
-    // Change the URL every 10 minutes to avoid caching.
-    FILETIME ft;
-    GetSystemTimeAsFileTime(&ft);
-    ULARGE_INTEGER uli{
-        .LowPart = ft.dwLowDateTime,
-        .HighPart = ft.dwHighDateTime,
-    };
-    uli.QuadPart /= 10000000ULL * 60 * 10;
-    return std::to_wstring(uli.QuadPart);
-}
```

Comment:
You seem to have replaced the code with an older version with your modifications. Please rebase and restore the removed/changed code which is not related to your PR.